### PR TITLE
Remove extra indentation of list help

### LINKED ISSUE #1345: Reduce excessive indentation of `limactl ls --help` output
I just noticed that the `limactl ls --help` output is badly formatted:

```console
$ limactl ls --help
List instances of Lima.
		The output can be presented in one of several formats, using the --format <format> flag.

		  --format json  - output in json format
		  --format yaml  - output in yaml format
		  --format table - output in table format
		  --format '{{ <go template> }}' - if the format begins and ends with '{{ }}', then it is used as a go template.

		These functions are available to go templates:

		  indent <size>: add spaces to beginning of each line
		  missing <message>: return message if the text is empty

		The following legacy flags continue to function:
		  --json - equal to '--format json'

Usage:
  limactl list [flags] [INSTANCE]...

Aliases:
  list, ls

Flags:
  -f, --format string   output format, one of: json, yaml, table, go-template (default "table")
  -h, --help            help for list
      --json            JSONify output
      --list-fields     List fields available for format
  -q, --quiet           Only show names

Global Flags:
      --debug   debug mode
```

The first block of text (before the actual "Usage:") is indented by 16 spaces (2 tabs), whereas it should be indented just 2 spaces.

_Originally posted by @jandubois in https://github.com/lima-vm/lima/pull/1267#pullrequestreview-1274627014_

### PR DESCRIPTION
**Pull request by afbjorklund**
Closes #1345

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -52,6 +52,7 @@ require (
 	github.com/insomniacslk/dhcp v0.0.0-20220504074936-1ca156eafb9f // indirect
 	github.com/kballard/go-shellquote v0.0.0-20180428030007-95032a82bc51 // indirect
 	github.com/kr/fs v0.1.0 // indirect
+	github.com/lithammer/dedent v1.1.0 // indirect
```
Comment (AkihiroSuda) [inline go.mod:55]: Doesn't seem indirect

**Review by jandubois — COMMENTED** [review]
Thanks, LGTM

File: go.mod (hunk 1 above)
Comment (afbjorklund) [reply go.mod:55]: go mod failed me.

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**